### PR TITLE
Get request for all of a users packing lists details

### LINKED ISSUE #25: Get request to return all user package lists
- The endpoint returns all packing lists associated with the user

### PR DESCRIPTION
**Pull request by Eternal-Flame085**
## Description
- Created tests for the new endpoint to get all of a user's packing lists
- Created a new  GET endpoint that returns all of a user's packing lists
- Removed unnecessary Travis run commands for heroku

Closes #25

### CODE REVIEW

File: api/resources/packing_list.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "num_of_days": packing_list.num_of_days,
+    "destination": packing_list.destination
+  }
+
+class PackingListResource(Resource):
+
+  def get(self, user_id):
+    if bool(db.session.query(Users).filter(Users.id == user_id).first()):
+        packing_lists = db.session.query(PackingLists).filter(PackingLists.user_id==user_id).all()
+        packing_lists_formatted = []
```
Comment (kevxo) [inline api/resources/packing_list.py:30]: Awesome stuff here

File: tests/resources/get_all_user_packing_lists_test.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    db.session.remove()
+    db_drop_everything(db)
+    self.app_context.pop()
+
+
+  def test_it_returns_a_list_of_items(self):
+    user_2 = Users(username='CrimsonGhost')
+    user_2.insert()
+
+    packing_list_1 = PackingLists(list_title='To Mars', user_id=1, num_of_days=6, destination="Mars")
```
Comment (kevxo) [inline tests/resources/get_all_user_packing_lists_test.py:31]: Nice setup!

**Review by kevxo — APPROVED** [review]
Looks good!

**Review by JoseLopez235 — APPROVED** [review]
LGTM

**Pull request merged**